app/models/criteria/association_builder.rb

```diff
@@ -19,19 +20,19 @@ def aggregate_plans(conditions)
         conditions.inject(AssociationPlanSet.new) { |p, c| c.prepare(p) }
       end
 
-      def finalize_plans(plans)
-        unless plans.planned_inner_join?(:completed_task)
-          plans.add(completed_task: :left_outer)
-        end
+      #def finalize_plans(plans)
+      #  unless plans.planned_inner_join?(:taggings)
+      #    plans.add(taggings: :left_outer)
+      #  end
 
-        unless plans.planned_inner_join?(:taggings)
-          plans.add(taggings: :left_outer)
-        end
+      #  unless plans.planned_inner_join?(:completed_task)
+      #    plans.add(completed_task: :left_outer)
+      #  end
 
-        if plans.planned_inner_join?(:completed_task) &&
-          plans.planned_left_outer_join?(:completed_task)
-          plans.delete(completed_task: :left_outer)
-        end
-      end
+      #  if plans.planned_inner_join?(:completed_task) &&
+      #    plans.planned_left_outer_join?(:completed_task)
+      #    plans.delete(completed_task: :left_outer)
+      #  end
+      #end
```

Comment:
Style/CommentIndentation: Incorrect indentation detected (column 6 instead of 4).<br>Style/LeadingCommentSpace: Missing space after #.